### ISSUE TITLE
CWD Tilde handling

### ISSUE THREAD

**Issue by vyeevani** [WebFreak001/code-debug#287]
Not sure if this is necessary a feature or just a fix for some so I'll classify it as a feature for now. Fail to find "lldb-mi" when the cwd has a "~" expansion in it.

**Comment by vyeevani** [comment]
According to this, the child process developers rejected handling cwd tilde expansion so it would have to be handled inside this repo: https://github.com/nodejs/node/issues/684

**Comment by GitMensch** [comment]
The feature implementation like that sounds easy:

```js
prog = prog.replace(/^~/, os.homedir());
```

**Comment by vyeevani** [comment]
I’ve made this PR to resolve this with untildify so that the support isn’t hard coded here as a string replacement. https://github.com/WebFreak001/code-debug/pull/288

**Comment by vyeevani** [comment]
@GitMensch do you happen to have write access to get this merged?

**Comment by GitMensch** [comment]
@vyeevani yes, but I won't do it now as @WebFreak001 had more or less the same questions that I had, please see the PR comments.
